src/Microsoft.DotNet.Interactive.ExtensionLab/SqlKernelUtils.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.DotNet.Interactive.ExtensionLab
+{
+    public static class SqlKernelUtils
```

Comment:
This should be kept `internal`.

Reply:
It wouldn't be visible to the test project then. Is that alright with you?

Reply:
That's ideal. It's an implementation detail and shouldn't be tested directly.